### PR TITLE
Dropdown

### PR DESCRIPTION
**Pull request by wereHamster**
Using downshift was a pretty pleasant experience.

image

### CODE REVIEW

File: src/components/Form/Dropdown.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    position: 'relative',
+    height: 40 + 20,
+    marginBottom: 12
+  }),
+  inner: css({
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    right: 0,
+    margin: '0 -12px 0 -12px',
```
Comment (uxengine) [inline src/components/Form/Dropdown.js]: Optional:
Could be shortened to
```
margin: '0 -12px',
padding: '0 12px',
```

File: src/components/Form/Dropdown.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+import zIndex from '../../theme/zIndex'
+import colors from '../../theme/colors'
+import {sansSerifRegular21} from '../Typography/styles'
+import {Label, LButton} from './Label'
+
+const styles = {
+  root: css({
+    position: 'relative',
+    height: 40 + 20,
```
Comment (uxengine) [inline src/components/Form/Dropdown.js]: Directly use labelHeight and fieldHeight from Label.js and export there?

File: src/index.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ import {simulations, speedy, css} from 'glamor'
 import theme from './catalogTheme'
 import './global.css'
 import './catalogTheme.css'
-import {fontStyles} from './components/Typography'
+import * as fontStyles from './components/Typography/styles'
```
Comment (wereHamster) [inline src/index.js]: Why was this necessary?

Comment (tpreusse) [reply src/index.js]: can't set `speedy(false)` after `css` from `glamor` has been used.  `Typography/index` uses it
